ToDoApp/Controllers/MainViewController.swift

```diff
@@ -6,46 +6,89 @@
 //
 import UIKit
 
-class MainViewController: UIViewController {
 
+
+final class MainViewController: UIViewController, AddItemDelegate {
+    // TableView에 사용될 데이터 모델
+    var items = [Info]()
+    
+    // MARK: - tableView 설정
+    let tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.rowHeight = 120
+        return tableView
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
         
-        self.title = "홈"
+        self.title = "ToDoList"
         self.view.backgroundColor = .white
+    
+  
+        // TableView 설정
+        tableView.register(CustomCell.self, forCellReuseIdentifier: CustomCell.cellId)
+        tableView.delegate = self
+        tableView.dataSource = self
+        view.addSubview(tableView)
         
-        // 네비게이션 아이템 추가
-        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "person.fill"), style: .plain, target: self, action: #selector(goToProfileVC))
+        // TableView Layout 설정
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top)
+            make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom)
+            make.leading.equalTo(view.safeAreaLayoutGuide.snp.leading)
+            make.trailing.equalTo(view.safeAreaLayoutGuide.snp.trailing)
```

Comment:
```suggestion
            make.edges.equalTo(view.safeAreaLayoutGuide)
```

로 단축하면 좋을 것 같아

`directionalHorizontalEdges`랑 `directionalVerticalEdges`도 기억해두면 편할듯!